app/views/projects/_index_table.html.erb

```diff
@@ -1,32 +1,34 @@
-<% if @projects.empty? %>
-  <p> No Project to show...</p>
+<% if projects.empty? %>
+  <p> <%= t('projects.no_project_to_show') %></p>
 <% else %>
   <br><br>
   <table class="table text-center">
-    <thead class="thead-dark">
+    <thead class="thead-light">
       <tr>
-        <th><%= t('projects.name_field') %></th>
-        <th><%= t('projects.type_field') %></th>
-        <th><%= t('projects.created_field') %></th>
-        <th><%= t('projects.actions_field') %></th>
+        <th><%= t('shared.sq_num_field') %></th>
+        <th><%= t('shared.name_field') %></th>
+        <th><%= t('shared.type_field') %></th>
+        <th><%= t('shared.created_field') %></th>
+        <th><%= t('shared.actions_field') %></th>
       </tr>
     </thead>
     <tbody>
-      <% @projects.each do |project|%>
+      <% projects.each.with_index(1) do |project,index|%>
```

Comment:
Index is not same as sequence number. Use sequence number like project.sequence_num
and column name say Project #